content/blog/tidymodels-2023-q4/index.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2023-q4
+title: "Q4 2023 tidymodels digest"
+date: 2024-01-08
+author: Emil Hvitfeldt
+description: >
+    The tidymodels team has been busy working on all sorts of new features 
+    across the ecosystem.
+photo:
+  url: https://www.pexels.com/photo/landscape-photography-of-snow-pathway-between-trees-during-winter-688660/
```

Comment:
Love the picture. :)

Reply:
thank you! it was the first thing that popped up when i searched for "winter" 😆  https://www.pexels.com/search/winter/